index.html

```diff
@@ -1082,8 +1090,15 @@
             <li>Set the <a>presentation identifier</a> of <var>S</var> to <var>
               I</var>.
             </li>
-            <li>Set the <a>presentation URL</a> of <var>S</var> to
-            <var>presentationUrl</var>.
+            <li>For each <var>presentationUrl</var> in
+            <var>presentationUrls</var> in order, run the following step:
+              <ol>
+                <li>If there is an entry <var>(presentationUrl, D)</var> in the
+                <a>list of available presentation displays</a>, then set the
+                <a>presentation URL</a> for <var>S</var> to
+                <var>presentationUrl</var> and stop iterating.
+                </li>
+              </ol>
```

Comment:
The above algorithm looks correct, but it's not obvious when you read it that it will **always** set the presentationUrl of `S`. Could the following make it clearer?

```
<li>Set the <a>presentation URL</a> for <var>S</var> to the first
<var>presentationUrl</var> in <var>presentationUrls</var> for which
there exists an entry <var>(presentationUrl, D)</var> in the <a>list of
available presentation displays</a>.
</li>
```